spec/models/user_spec.rb

```diff
@@ -49,4 +49,20 @@
       expect(user.admin?).to be_truthy
     end
   end
+
+  describe "reassign_topics_to_new_users" do
+    let(:other_user) { create(:user) }
+    let(:my_topic) { create(:topic, user: user) }
+    let(:other_user_bookmark) { create(:bookmark, topic: my_topic, user: other_user) }
+
+    it "should reassign a topic to other_user" do
+      delete :destroy, {id: user.id}
+      expect(assigns(:user)).to eq(other_user)
+    end
+    it "should delete a topic if there is no user to reassign it to" do
+      delete :destroy, {id: user.id}
+      count = User.where({id: user.id}).size
```

Comment:
ok, a few things:
first of all, it looks like these are controller specs, but this is in a model spec file.  so we need to either move these to the users_controller_spec file or change them to use syntax that isn't testing a controller.

also, it looks like you're testing that there are no users with a given id.  there are better ways to test this.  you can get the count of users before you destroy the user and then test that the count decreased by 1.  you can just try to find this user (`User.find(user.id)`) and expect it to be nil.  you can use a `change` expectation to check that the count changed by 1.  lots of options.